tests/unit/common/test_config.py

```diff
@@ -0,0 +1,126 @@
+import unittest
+
+import os
+
+from ims.common import config
+
+config.load()
+from ims.exception import config_exceptions
+from ims.common.log import trace
+
+
+class TestBadPath(unittest.TestCase):
+    """ Tests whether an exception is raised when loading from invalid path """
+
+    @trace
+    def setUp(self):
+        self.old_path = os.getenv('BMI_CONFIG')
+        os.environ['BMI_CONFIG'] = "blahblah.cfg"
+
+    def runTest(self):
+        with self.assertRaises(IOError):
+            config.load(force=True)
+
+    def tearDown(self):
+        os.environ['BMI_CONFIG'] = self.old_path
+        config.load(force=True)
+
```

Comment:
Ok, if you have force as bool variable in code, you should check both in test cases, if you can move the config in some other function and start calling the function after this has been loaded, will there be a problem?